src/event.js

```diff
@@ -677,30 +683,24 @@ jQuery.Event.prototype = {
 	isImmediatePropagationStopped: returnFalse,
 
 	preventDefault: function() {
-		var e = this.originalEvent;
-
 		this.isDefaultPrevented = returnTrue;
 
-		if ( e && e.preventDefault ) {
-			e.preventDefault();
+		if ( this.originalEvent ) {
```

Comment:
Also it would be more consistent with native events, event in those do the nop when `preventDefault` is not possible, not error